TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift

```diff
@@ -57,7 +58,7 @@ class PushFactory {
 
 extension PushFactory: PushFactoryProtocol {
   func createFactor(withAccessToken accessToken: String, friendlyName: String, serviceSid: String,
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/8493b62acdbfcf89b7484fe450452cd8fa9157ed/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L60"></span>Function should have 7 parameters or less: it currently has 8<br />
<code>function_parameter_count</code> <code>PushFactory.swift:60</code></td>
    </tr>
  </tbody>
</table>